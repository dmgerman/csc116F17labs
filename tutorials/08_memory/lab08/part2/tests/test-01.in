Tokyo
Ottawa
Washington
London
Madrid
Amsterdam
Mexico City
Stockholm
