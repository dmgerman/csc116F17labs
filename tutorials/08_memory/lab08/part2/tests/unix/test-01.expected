Capitals:
Tokyo -> Ottawa -> Washington -> London -> Madrid -> Amsterdam -> Mexico City -> Stockholm

is Ottawa in the list? true
is Brazilia in the list? false
Print empty list:

is anything in the list? false
